README.md

```diff
@@ -249,6 +254,13 @@ query ($id: String!) {
 
 <!-- TODO add screenshot -->
```

Comment:
is this a screenshot of the site or what?

Reply:
I think I meant this for the code snippet that was added above
```suggestion
```